src/manifest.c

```diff
@@ -632,12 +637,13 @@ struct manifest *load_manifest(int current, int version, struct file *file, stru
 		return NULL;
 	}
 
-	ret = verify_bundle_hash(mom, file);
+	if (!header_only)
```

Comment:
also, because `header_only` is not a bool or pointer, it would be more clear to use an explicit integer comparison.